engine/pom.xml

```diff
@@ -206,7 +208,6 @@
                             <classesDirectory>${project.build.testOutputDirectory}</classesDirectory>
                             <includes>
                                 <include>org/hibernate/validator/testutil/**</include>
-                                <include>org/hibernate/validator/test/internal/metadata/provider/AnnotationMetaDataProviderTestBase*</include>
```

Comment:
This class is used in the new engine-jdk8-tests module.
You should have some error if you remove it.